src/Microsoft.AspNetCore.Mvc.Core/ApplicationParts/AssemblyPart.cs

```diff
@@ -40,7 +40,7 @@ public AssemblyPart(Assembly assembly)
         public IEnumerable<TypeInfo> Types => Assembly.DefinedTypes;
 
         /// <inheritdoc />
-        public IEnumerable<string> GetReferencePaths()
+        public virtual IEnumerable<string> GetReferencePaths()
```

Comment:
This is a breaking change.

Reply:
I believe it's not, I remember @Eilon told me the compiler already emits callvirt for these things no matter what.

Reply:
And this behavior is encoded on API check

Reply:
Well, compilers *at the time* did that 😄 But I think we still consider it safe.

Reply:
Yeah we talked about this. I'm fairly certain it's fine

Reply:
I'm changing my mind. The .NET team recently changed their guidance, per this PR: https://github.com/dotnet/corefx/pull/20403/files

So, this is a more serious breaking change these days than it used to be.